src/components/WwwFrame/TheHeader.vue

```diff
@@ -618,356 +614,43 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import 'settings';
-
-$top-nav-font-size: 1.125rem;
-$header-height: rem-calc(45);
-$header-height-large: rem-calc(64);
-$header-background-color: $white;
-$header-text-color: $link-gray;
-$header-logo-color: $kiva-green;
-$header-link-color: $link-gray;
-$header-link-hover-color: $platinum;
-$header-separator-color: $white;
-$form-padding: 0.32rem;
-$form-padding-large: 0.6rem;
-$close-search-button-size: 2.5rem;
-
-.top-nav {
-	background: $header-background-color; // IE11 fallback
-	background: var(--kv-header-background-color, $header-background-color);
-	color: $header-text-color; // IE11 fallback
-	color: var(--kv-header-text-color, $header-text-color);
-	font-size: $top-nav-font-size;
-	font-weight: $global-weight-highlight;
-	box-shadow: var(--kv-header-shadow, $navbar-shadow);
-	z-index: 1000;
-
-	.amount {
-		@include breakpoint(large) {
-			color: $header-background-color; // IE11 fallback
-			color: var(--kv-header-background-color, $header-background-color);
-			background: $header-link-color; // IE11 fallback
-			background: var(--kv-header-link-color, $header-link-color);
-			padding: rem-calc(1) rem-calc(7);
-		}
-	}
-
-	.dropdown-pane {
-		border-top: none;
-		font-size: 1rem;
-
-		ul {
-			margin: 0;
-		}
-
-		li {
-			list-style: none;
-		}
-
-		@include breakpoint(medium down) {
-			width: 100%;
-			left: 0 !important;
-
-			button,
-			a,
-			li > span {
-				display: block;
-				width: 100%;
-				padding: 0.5rem 1rem;
-				border-bottom: 1px solid $kiva-stroke-gray;
-			}
-		}
-	}
-
-	.dropdown-list {
-		a {
-			display: block;
-			width: 100%;
-			padding: 0.5rem 1rem;
-			white-space: nowrap;
-			border-bottom: 1px solid $kiva-stroke-gray;
-
-			@include breakpoint(large) {
-				border-bottom: none;
-			}
-		}
-
-		hr {
-			display: none;
-			margin: 0 1rem;
-
-			@include breakpoint(large) {
-				display: block;
-			}
-		}
-	}
+<style lang="postcss" scoped>
+.header__button {
+	@apply tw-flex tw-items-center tw-flex-shrink-0;
+	@apply tw-text-primary hover:tw-text-action-highlight hover:tw-no-underline focus:tw-no-underline;
+	@apply tw-h-8 md:tw-h-9 tw-whitespace-nowrap tw-flex-shrink-0;
```

Comment:
I went with 64 mobile, 72 tablet since those are values in our design system. And I couldn't rationalize why tablet would have a smaller nav than desktop.

Reply:
We're going with 64 on mobile and 72 for medium and desktop as those values fit our sizing scale.